pkg/gui/branches_panel.go

```diff
@@ -62,20 +62,28 @@ func (gui *Gui) handleNewBranch(g *gocui.Gui, v *gocui.View) error {
 }
 
 func (gui *Gui) handleDeleteBranch(g *gocui.Gui, v *gocui.View) error {
+	gui.deleteBranch(g, v, false)
+}
+
+func (gui *Gui) handleForceDeleteBranch(g *gocui.Gui, v *gocui.View) error {
+	gui.deleteBranch(g, v, true)
+}
+
+func (gui *Gui) deleteBranch(g *gocui.Gui, v *gocui.View, force bool) error {
 	checkedOutBranch := gui.State.Branches[0]
 	selectedBranch := gui.getSelectedBranch(v)
 	if checkedOutBranch.Name == selectedBranch.Name {
 		return gui.createErrorPanel(g, gui.Tr.SLocalize("CantDeleteCheckOutBranch"))
 	}
+	title := gui.Tr.SLocalize("DeleteBranch")
```

Comment:
We should add 'force delete' to the message if force == true, so that it's clear to the user whether they are deleting the branch locally or also remotely.

Reply:
I can adjust the message, no problem. But I think you are unclear on the purpose of force. It is not about local vs remote;

```
  -d, --delete
      Delete a branch. The branch must be fully merged in its 
      upstream branch, or in HEAD if no upstream was set with 
      --track or --set-upstream-to.

  -f, --force
      Reset <branchname> to <startpoint>, even if <branchname>
      exists already. Without -f, git branch refuses to change
      an existing branch. In combination with -d (or --delete),
      allow deleting the branch irrespective of its merged
      status. In combination with -m (or --move), allow renaming
      the branch even if the new branch name already exists,
      the same applies for -c (or --copy).
```

Reply:
It allows you to locally delete a branch even if it is not considered fully merged yet (which for me is often the case since I frequently rebase).

Reply:
ah cool thanks for the explanation :)

Let me know when the PR is ready for re-review